coordinator/.env.example

```diff
@@ -1,2 +1,22 @@
+# Rate limit configuation
 TTL=60000
 LIMIT=10
+
+# Make sure your private and public RSA keys are generated (see package.json scripts)
+# Public key must be copied to frontend app to encrypt user sensitive data
```

Comment:
what user data? are you referring to coordinator's key?

Reply:
Right, for now it's only coordinator private key.